packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -82,7 +82,7 @@
             transform-origin: top left;
 
             &-text {
-                font-size: 1.4em;
+                font-size: 1.4rem;
```

Comment:
`var(--text-size-xs: 1.4rem)`

Reply:
```suggestion
                font-size: var(--text-size-xs: 1.4rem);
```

Reply:
We tend to do this in most other places as well.
Probably best to do this in a `font-size` refactoring card altogether.